docs_user/modules/proc_adopting-networker-services-to-the-data-plane.adoc

```diff
@@ -2,34 +2,18 @@
 
 = Adopting Networker services to the {rhos_acro} data plane
 
-If you have `Networker` nodes in your existing {rhos_acro} deployment, you
```

Comment:
This section looks to have a lot of "you might need to do this" and "you might need to do that" instead of actual narrow prescribed directions like "only do this" and "only do that" and stay in your lane and don't worry about any of that other stuff. I wonder is this conducive to more customer feedback / support cases or it's simply the nature of the technology and unavoidable?